addons/core/functions/fnc_hasRadio.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+
+/*
+    Name: TFAR_fnc_hasRadio
+
+    Author(s):
+        Dorbedo
+
+    Description:
+        Check if a unit has a radio
+
+    Parameters:
+        0: OBJECT - the unit
+
+    Returns:
+        the unit has a radio
+
+    Example:
+    _hasRadio = [_player] call TFAR_fnc_hasRadio;
+ */
+ 
+params [["_unit", objNull, [objNull]]];
+
+If !(_unit isKindOf "CAManBase") exitWith {false};
+
+(((count ([_unit] call TFAR_fnc_getRadioItems))>0)
+||!((_unit call TFAR_fnc_backpackLr) isEqualTo []))
```

Comment:
`|| {!((_unit call TFAR_fnc_backpackLr) isEqualTo [])}`
lazy eval.
I would even say execute backpackLR first because thats the cheaper one.